### DISCUSSION TITLE
Ist die Verwendung von Bootstrap 5.x schon in der Pipeline?

### DISCUSSION THREAD

**Discussion by schroeder-markus** [affiliatetheme/affiliatetheme#15] [General]
Hi,

ist Bootstrap 5.x schon in der Pipeline?

VG
Markus

**Comment by StrongLucky** [comment]
Hallo Markus,

es steht zumindest auf der Liste ;)

Beste Grüße
Lucas

**Comment by endcoreAK** [comment]
Kurz ergänzend, zwischen 5.2 und 5.3 gab es massive Änderungen, sodass 5.3 theoretisch als 6.0 durchgehen könnte. Daher macht es aktuell noch keinen Sinn die Entwicklung für 5.x zu fokussieren bzw. gar anzufangen, bevor diese in eine Stable draussen ist. :)
